pkg/linguist/dev.go

```diff
@@ -0,0 +1,84 @@
+package linguist
+
+import (
+	"strings"
+
+	"github.com/okteto/cnd/pkg/model"
+	log "github.com/sirupsen/logrus"
+)
+
+type languageDefault struct {
+	image   string
+	command []string
+	path    string
+}
+
+const (
+	javascript       = "javascript"
+	golang           = "go"
+	python           = "python"
+	unrecognized     = "unrecognized"
+	helloCommandName = "hello"
+)
+
+var (
+	languageDefaults map[string]languageDefault
+	tailCommand      = []string{"tail", "-f", "/dev/null"}
+)
+
+func init() {
+	languageDefaults = make(map[string]languageDefault)
+	languageDefaults[javascript] = languageDefault{
+		image:   "node",
+		command: tailCommand,
```

Comment:
100% of node projects will have to do `npm install` first and a very high percentage of them will then do `npm start` which is the default script to execute. I would use `npm install & npm start` as default command here.

`npm` is the official and more established package manager, but `yarn` is the new trend and it is faster. Still not sure which one we should use though.